.github/workflows/build-plugin/action.yml

```diff
@@ -22,7 +22,13 @@ runs:
                 echo "PACKAGE_VERSION is empty string or undefined"
                 exit 1
             fi
-            bash ${{ inputs.UPDATE_VERSIONS_SCRIPT_PATH }}
+
+            if [ -f "${{ inputs.UPDATE_VERSIONS_SCRIPT_PATH }}" ];
+              then
+               bash ${{ inputs.UPDATE_VERSIONS_SCRIPT_PATH }}
+            fi
```

Comment:
When Cloud publish begins, it will run from the tag.
By updating the tag's version first, then running the cloud publish action, the version will increase twice.

If we take the current version x.5 as an example
This will increase the tag version by one, resulting in x.6.
As soon as the publish cloud runs, it will increase it again, but this time from x.6 to x.7.

Reply:
Try to make it explicit (e.g., `create_new_version`=true|false), not as guessing the logic based on if a file is existing or not.